modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/headless/batch/engine/resource/v1_0/test/ImportTaskResourceTest.java

```diff
@@ -31,6 +40,36 @@
 @RunWith(Arquillian.class)
 public class ImportTaskResourceTest extends BaseTaskResourceTestCase {
 
+	@Test
+	public void testDeleteImportTask() throws Exception {
+		ObjectEntry objectEntry = ObjectEntryTestUtil.addObjectEntry(
+			objectDefinition, OBJECT_FIELD_NAME_TEXT, "TestObject");
+
+		int objectEntriesCount = ObjectEntryTestUtil.getObjectEntriesCount();
+
+		ImportTaskResource importTaskResource = ImportTaskResource.builder(
+		).authentication(
+			"test@liferay.com", PropsValues.DEFAULT_ADMIN_PASSWORD
+		).header(
+			HttpHeaders.ACCEPT, ContentTypes.APPLICATION_JSON
+		).header(
+			HttpHeaders.CONTENT_TYPE, ContentTypes.APPLICATION_JSON
+		).parameter(
+			"siteId", String.valueOf(TestPropsValues.getGroupId())
+		).build();
+
+		String payload = JSONUtil.putAll(
+			JSONUtil.put(
+				"externalReferenceCode", objectEntry.getExternalReferenceCode())
+		).toString();
+
+		_executeDeleteImportTask(importTaskResource, payload);
+
+		Assert.assertEquals(
+			ObjectEntryTestUtil.getObjectEntriesCount(),
+			objectEntriesCount - 1);
```

Comment:
objectsCount is not a bad idea, but I wonder if we could just make a call in order to make sure then when calling for the particular objectEntry it is 404 NOT_FOUND